src/etc/inc/xmlparse.inc

```diff
@@ -248,7 +248,8 @@ function dump_xml_config_sub($arr, $indent) {
 						    (substr($ent, 0, 9) == "ldap_bind") ||
 						    (substr($ent, 0, 11) == "ldap_basedn") ||
 						    (substr($ent, 0, 18) == "ldap_authcn") ||
-						    (substr($ent, 0, 19) == "ldap_extended_query")) {
+						    (substr($ent, 0, 19) == "ldap_extended_query") ||
+							($ent == "customwidgettitle")) {
```

Comment:
Now that I am aware of this stuff embedded in the XML parser, that seems a good idea. The "descr" key will be a sub-key of each individual widget anyway, so the "descr" key name will seem reasonable for being put in the "widget title".